### PR TITLE
Add functionality to get the invoice in store.

### PR DESCRIPTION
**Pull request by chamindias**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import MUIDataTable from "mui-datatables";
+
+import { FormattedMessage } from 'react-intl';
+import Subscription from 'AppData/Subscription';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        padding: theme.spacing(3, 2),
+    },
+}));
```
Comment (chanaka3d) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx]: We can remove this

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        <React.Fragment>
+            <Button
+                variant = 'outlined'
+                size = 'small'
+                color = 'primary'
+                disabled={!(isMonetizedAPI && isDynamicUsagePolicy)}
+                onClick = {handlePopup}
+            >
+                <FormattedMessage
+                    id = 'Applications.Details.SubscriptionTableData.view.subscription.invoice'
```
Comment (chanaka3d) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx]: id have to be something like Applications.Details.Invoice.view.btn

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx (hunk 1 above)
Comment (chamindias) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx]: Done : https://github.com/wso2/carbon-apimgt/pull/7306/commits/d20d664c58b0dfde9632423e3db399e740aaadfa

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx (hunk 2 above)
Comment (chamindias) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/Details/Invoice.jsx]: Done : https://github.com/wso2/carbon-apimgt/pull/7306/commits/d20d664c58b0dfde9632423e3db399e740aaadfa

**Comment by chamindias** [comment]
Related to : https://github.com/wso2/product-apim/issues/4680

**Pull request merged**